sekoia_automation/trigger.py

```diff
@@ -36,6 +38,23 @@ def __init__(self, module: Module | None = None, data_path: Path | None = None):
         self._configuration: dict | BaseModel | None = None
         self._data_path = data_path or get_data_path()
         self._error_count = 0
+        self._secrets: dict[str, Any] = {}
+
+    def _get_secrets(self) -> dict[str, Any]:
```

Comment:
This method should be called only if module has secrets, to avoid sending a useless query.

Reply:
done